File: challenge-01/main.py
response = ''
map = dict()
with open('challenge-01/data.txt','r') as f:
    for word in f.readline().split(' '):
        if word in map:
            map[word]+=1
            continue
        map[word]=1
for word in map:
    response+=word+str(map[word])
print (response)

File: challenge-01/main.dart
import 'dart:io';

void main() {
  File('challenge-01/data.txt').readAsString().then((String data) {
    final hashMap = {};
    for (final word in data.split(' '))
      hashMap.containsKey(word) ? hashMap[word] += 1 : hashMap[word] = 1;
    final response = hashMap.entries.fold<String>(
        '', (previous, next) => '$previous${next.key}${next.value}');
    print(response);
  }).onError((_, __) {
    print('archivo inexistente');
  });
}
